nbviewer/app.py

```diff
@@ -281,6 +282,7 @@ def init_options():
     define("debug", default=False, help="run in debug mode", type=bool)
     define("no_cache", default=False, help="Do not cache results", type=bool)
     define("localfiles", default="", help="Allow to serve local files under /localfile/* this can be a security risk", type=str)
+    define("localfile_abspath", default=False, help="Use the absolute instead of the real path for local files, don't dereference symbolic links", type=bool)
```

Comment:
Minor suggestion: Maybe name the parameter `localfile_follow_symlinks`. Personally, I always need to google to remember which one of realpath or abspath dereferences the symlinks.

Reply:
Nice suggestion, I'm not great at naming stuff. ;-)

Will do.